include/gadget/sockets-map.h

```diff
@@ -245,4 +246,21 @@ gadget_socket_lookup(const struct sock *sk, __u32 netns)
 }
 #endif
 
+static __always_inline void
+gadget_socket_fill_process(const struct sockets_value *skb_val, struct gadget_process *p) {
```

Comment:
Should the definition of `struct sockets_value` be changed to include `struct gadget_process` as an inner field?

I guess it's better to avoid the change because that would break the compatibility between ig version n+1 and gadgets compiled with ig version n.

Reply:
That's the reason why I explicitly avoided to touch that structure. But I think in the long term it'll better to embed this structure there to keep them as aligned as possible.